packages/ui/src/components/templates/report/components/AdsAndSocialMedia/AdsAndSocialMedia.tsx

```
@@ -1,104 +1,236 @@
+      <h3 className="mb-2 text-base font-bold">Social Media</h3>
+
+      <div className="flex w-full flex-col gap-4">
+        {AdsProviders.map(toLowerCase).map(provider => {
+          const { page, ...rest } = mediaPresence[provider] ?? {};
+          const { src, link } = adsImages[provider] ?? {};
+
+          // || because empty string is not a valid case
+          const idValue = ('id' in rest ? rest.id : rest.userName) || null;
+
+          return (
+            <Card className={ctw('w-full p-4 shadow-l', !page && 'opacity-60')}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)